### PR TITLE
Robotic lungs can now be configured for vox and plasmamen

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Allows cybernetic and upgraded cybernetic lungs to be configured to vox and plasmamen lung settings. The upgraded lungs will also give vox 20 oxygen as the max oxygen level, similar to the toxin and co2 bonus upgraded lungs already give, but it is **NOT** enough for them to safely breath station oxygen. It will however lower the toxin damage to 1 per cycle.

## Why It's Good For The Game

Being able to properly implant a species lungs into them when no organic replacements are available is good. Currently, nothing stops you from putting robotic lungs into a vox / plasmamen... which ends up being silly. Replacing someones lungs should not magically make them breath a different atmosphere. As such, this PR lets robotic and upgraded lungs be configured to plasmamen and vox settings with a multitool, so they still breath their gas after getting their lungs replaced.

## Changelog
:cl:
add: Cybernetic and upgraded cybernetic lungs can be configured with a multitool for vox or plasmamen settings.
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/lungs.dm [hunk 1]
```diff
@@ -345,6 +345,39 @@
...
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	switch(species_state)
+		if("human")
+			safe_oxygen_min = 0
+			safe_oxygen_max = vox_oxygen_bonus
+			safe_nitro_min = 16
+			oxy_damage_type = TOX
+			user.show_message("You configure [src] to replace vox lungs.")
```
Comment (hal9000PR) [inline code/modules/surgery/organs/lungs.dm:365]: please add a span class for these. and, why are you using show_message instead of to_chat?

**Comment by procdrone** [comment]
time for me to replace my lungs with vox cyber lungs, then have officers baited into manslaughter as they search me in sec and remove my voxxygen :star:

File: code/modules/surgery/organs/lungs.dm [hunk 2]
```diff
@@ -345,6 +345,38 @@
...
+/obj/item/organ/internal/lungs/cybernetic/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>[src] is configured for [species_state] standards of atmosphere.</span>"
+
+/obj/item/organ/internal/lungs/cybernetic/multitool_act(mob/user, obj/item/I)
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	switch(species_state)
+		if("human")
```
Comment (AffectedArc07) [inline code/modules/surgery/organs/lungs.dm:359]:
```suggestion
		if("human") // from human to vox
```

File: code/modules/surgery/organs/lungs.dm [hunk 3]
```diff
@@ -345,6 +345,38 @@
...
+			species_state = "vox"
+		if("vox")
+			safe_oxygen_max = initial(safe_oxygen_max)
+			safe_toxins_min = 16
+			safe_toxins_max = 0
+			safe_nitro_min = initial(safe_nitro_min)
+			oxy_damage_type = OXY
+			to_chat(user, "<span class='notice'>You configure [src] to replace plasmamen lungs.</span>")
+			species_state = "plasmamen"
+		if("plasmamen")
```
Comment (AffectedArc07) [inline code/modules/surgery/organs/lungs.dm:374]:
```suggestion
		if("plasmamen") // from plasmamen to human
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**